main.tf

```diff
@@ -405,6 +405,29 @@ resource "aws_codepipeline" "this" {
   depends_on = ["aws_iam_role_policy.codebuild", "aws_ecs_service.this"]
 }
 
+### CLOUDWATCH BASIC DASHBOARD
+
+data "template_file" "metric_dashboard" {
+  count = "${length(var.services) > 0 ? length(var.services) : 0}"
+
+  template = "${file("${path.module}/metrics/basic-dashboard.json")}"
+
+  vars {
+    region         = "${var.region}"
+    alb_arn_suffix = "${element(aws_lb.this.*.arn_suffix, count.index)}"
+    cluster_name   = "${aws_ecs_cluster.this.name}"
+    service_name   = "${element(keys(var.services), count.index)}"
+  }
+}
+
+resource "aws_cloudwatch_dashboard" "this" {
```

Comment:
Could `this` be renamed to something more meaningful, like `basic` (as per the comment above the template file)?